applications/StructuralMechanicsApplication/custom_python/structural_mechanics_python_application.cpp

```diff
@@ -120,14 +120,21 @@ PYBIND11_MODULE(KratosStructuralMechanicsApplication,m)
 
     // Membrane variables
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, MEMBRANE_PRESTRESS )
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, PRESTRESS_VECTOR )
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, PRESTRESS_AXIS_1_GLOBAL )
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, PRESTRESS_AXIS_2_GLOBAL )
-    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, PRESTRESS_AXIS_1_LOCAL )
-    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, PRESTRESS_AXIS_2_LOCAL )
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, PRESTRESS_AXIS_1 )
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, PRESTRESS_AXIS_2 )
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, PROJECTION_TYPE_COMBO )
 
     //Formfinding
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, LAMBDA_MAX )
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, IS_FORMFINDING )
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, BASE_REF_1 )
```

Comment:
What is this variable?

Reply:
base vector 1 in reference configuration